packages/react-components/src/menu/src/Menu.tsx

```diff
@@ -86,6 +86,10 @@ export interface InnerMenuProps extends DomProps, AriaLabelingProps, ComponentPr
      * React children.
      */
     children: ReactNode;
+    /**
+     * @ignore
+     */
+    disabled?: boolean;
```

Comment:
I don't think we still need to define attributes like "disabled" in our interface since they should be included in ComponentProps